packages/theme-chalk/src/common/var.scss

```diff
@@ -10,14 +10,27 @@
 
 /* Transition
 -------------------------- */
-$--all-transition: all 0.3s cubic-bezier(0.645, 0.045, 0.355, 1) !default;
-$--fade-transition: opacity 300ms cubic-bezier(0.23, 1, 0.32, 1) !default;
-$--fade-linear-transition: opacity 200ms linear !default;
-$--md-fade-transition: transform 300ms cubic-bezier(0.23, 1, 0.32, 1),
-  opacity 300ms cubic-bezier(0.23, 1, 0.32, 1) !default;
-$--border-transition-base: border-color 0.2s
-  cubic-bezier(0.645, 0.045, 0.355, 1) !default;
-$--color-transition-base: color 0.2s cubic-bezier(0.645, 0.045, 0.355, 1) !default;
+// refer to this website to get the bezier motion function detail
+//  https://cubic-bezier.com/#p1,p2,p3,p4 (change px as your function parameter)
+
+:root {
+  --el-transition-duration: 300ms;
+  --el-ez-in-out-bezier-function: cubic-bezier(0.645, 0.045, 0.355, 1);
+  --el-fast-bezier-transition: cubic-bezier(0.23, 1, 0.32, 1);
+}
+
+$--all-transition: all var(--el-transition-duration)
```

Comment:
Now that we have removed the variable, then we can use `--el-all-transition` instead of `$--all-transition`.

Reply:
Yup, will address this later.